app/scripts/lib/web3-entities.json

```diff
@@ -0,0 +1,101 @@
+{
+  "web3.bzz.blockNetworkRead": "function",
+  "web3.bzz.download": "function",
+  "web3.bzz.get": "function",
+  "web3.bzz.getHive": "function",
+  "web3.bzz.getInfo": "function",
+  "web3.bzz.hive": "TRAP",
+  "web3.bzz.info": "TRAP",
+  "web3.bzz.modify": "function",
+  "web3.bzz.put": "function",
+  "web3.bzz.retrieve": "function",
+  "web3.bzz.store": "function",
+  "web3.bzz.swapEnabled": "function",
+  "web3.bzz.syncEnabled": "function",
+  "web3.bzz.upload": "function",
+  "web3.db.getHex": "function",
+  "web3.db.getString": "function",
+  "web3.db.putHex": "function",
+  "web3.db.putString": "function",
+  "web3.eth.accounts": "object",
```

Comment:
Is it worth recording metrics for this at all? It's likely used for fingerprinting so much that we'll have to exclude it anyway to get any useful data.

Though I guess we have yet to confirm that :thinking:

Reply:
Yeah, I think your hunch is good, but we haven't confirmed it.

This would confirm it!